images/ha-config/keepalived/tests/echoserver.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
```

Comment:
Yeah, this was just to test and verify the image is okay before I had the integration code in place. The image verification test monitors the echo server listen port.